pkg/server/util/regentryutil/cache_test.go

```diff
@@ -0,0 +1,66 @@
+package regentryutil
+
+import (
+	"testing"
+	"time"
+
+	"github.com/spiffe/spire/proto/spire/common"
+	"github.com/stretchr/testify/require"
+)
+
+type mockTime struct {
```

Comment:
Can we reuse `clock.Mock` here?

https://github.com/spiffe/spire/blob/master/test/clock/clock.go

Reply:
Thanks, using this clock now